tests/plugins/inlinetoolbar/buttons.js

```diff
@@ -29,7 +29,18 @@
 			assert.isInstanceOf( CKEDITOR.ui.button, panel.getItem( 'cut' ), 'Registered button type.' );
 			panel.deleteItem( 'cut' );
 			assert.isUndefined( panel.getItem( 'cut' ), 'The button should be deleted.' );
+		},
 
+		'test group wrapping': function() {
+			var panel = new CKEDITOR.ui.inlineToolbar( this.editor );
+			panel.addItems( {
+				cut: new CKEDITOR.ui.button( {
+					label: 'test',
+					command: 'cut'
+				} )
+			} );
+			panel._view.renderItems( panel._items );
+			assert.isNotNull( panel._view.parts.content.findOne( '.cke_toolgroup' ), 'Button should be wrapped in group' );
```

Comment:
That's a good test to have, I'm missing one more. An unit test that will add both button and a rich combo, and then ensure that 2 `.cke_toolgoup` elements are created.